README.md

```diff
@@ -30,14 +30,27 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+            ForEach iterates oves an array and modifies it in the way specified within the function,  map iterates over an array and returns another, new array with altered items in some way specified in the function.
+
 2. What is the difference between a function and a method?
 
+             A method the value declared as the value to the key:value pair of/ within another function.  (A function within a function)
+
```

Comment:
Your answer appears correct. Take look at this answer as well: https://stackoverflow.com/questions/155609/whats-the-difference-between-a-method-and-a-function